src/appleseed-max-impl/utilities.cpp

```diff
@@ -174,14 +174,25 @@ bool is_osl_texture(Texmap* map)
     return dynamic_cast<OSLTexture*>(map) != nullptr;
 }
 
-bool is_supported_procedural_texture(Texmap* map)
+bool is_supported_procedural_texture(Texmap* map, bool osl_mode)
```

Comment:
Make keeping `use_max_procedural_maps` would be less confusing? (In which case the test must be negated since `osl_mode` = `!use_max_procedural_maps`).

Reply:
Agree, I'll change it to `use_max_procedural_maps`